pkgs/by-name/co/colmap/package.nix

```diff
@@ -22,71 +25,101 @@
   cudaSupport ? config.cudaSupport,
   cudaCapabilities ? cudaPackages.flags.cudaCapabilities,
   cudaPackages,
+  faiss,
+  llvmPackages,
+  gtest,
 }:
 
 assert cudaSupport -> cudaPackages != { };
 
 let
-  boost_static = boost.override { enableStatic = true; };
   stdenv' = if cudaSupport then cudaPackages.backendStdenv else stdenv;
 
+  # These deps are also needed by pycolmap.
+  pythonDeps = [
+    boost
+    eigen
+    ceres-solver
+    openimageio
+    glog
+    libGLU
+    glew
+    cgal
+    poselib
+    faiss
+    sqlite
+    gmp
+    mpfr
+    lz4
+    qt5.qtbase
+  ]
+  ++ lib.optionals cudaSupport [
+    cudatoolkit
+    cudaPackages.cuda_cudart.static
```

Comment:
While moving this code and in case you're interested in picking up maintainership of colmap:

- `cudaPackagages.cudatoolkit` is a `symlinkJoin` of a bunch of other things and we try to avoid it
- `lib.getStatic`

Reply:
Also a bit confusing to call these `pythonDeps`

Reply:
That all sounds good but I'm not sure how that works and it sounds like it would need upstream to switch to FindCudaToolkit. We can do it, but I'd like to get the package unbroken first.